tests/integration/world_io_test.py

```diff
@@ -1,8 +1,7 @@
 import unittest
 
-from graspit_commander.graspit_exceptions import (
-    LoadWorldException
-)
+from graspit_commander.graspit_exceptions import *
```

Comment:
We're trying to avoid using `import *`. It's better practice to use explicit declarations and it makes it easier to see what you're working with within the file. It can also be a good way to remember what's (imported) in the current local namespace.